Makefile

```diff
@@ -25,3 +26,4 @@ format-check:
 	(isort -rc $(PYTHON_FILES) --check-only --multi-line=3 --trailing-comma --force-grid-wrap=0 --use-parentheses --line-width=88) && (black -t py37 --check $(PYTHON_FILES)) || (echo "run \"make format\" to format the code"; exit 1)
 	pylint -j0 $(PYTHON_FILES)
 	mypy --show-error-codes $(PYTHON_FILES)
+	npx prettier $(CSS_FILES) --check
```

Comment:
this will check the CSS format
Also, I use `npx` to execute prettier so you don't need to add `./node_modules/.bin` to the `PATH`.

Reply:
nice.